src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs

```diff
@@ -882,6 +901,12 @@ protected virtual string GenerateDisplayText(ModelExplorer modelExplorer)
                 tagBuilder.WriteTo(ViewContext.Writer, _htmlEncoder);
             }
 
+            if (!suppressAntiforgery)
```

Comment:
Discussed and agreed to move it to a separate issue.https://github.com/aspnet/Mvc/issues/3868